src/Hit/Issue.hs

```diff
@@ -121,6 +123,44 @@ createIssue title login = withOwnerRepo $ \token owner repo -> case token of
         errorMessage errorText
         pure $ Left $ ParseError errorText
 
+{- | Assign the user to the given 'Issue'.
+
+This function can fail assignment due to the following reasons:
+
+ * Auth token fetch failure
+ * Assignment query to GutHub failure
+
+The function should inform user about corresponding 'Error' in each case and
+continue working.
+-}
+assignIssue :: Issue -> Text -> IO ()
+assignIssue issue username = do
+    res <- withOwnerRepo $ \token owner repo -> case token of
+        Just auth -> do
+            let assignee :: Name User
+                assignee = makeName @User username
+            let curAssignees :: V.Vector (Name User)
+                curAssignees = V.map simpleUserLogin $ issueAssignees issue
+
+            if V.elem assignee curAssignees
```

Comment:
There's `assignedTo` function above. Can we more your implementation on top-level and use in the function above? Or, simply, rewrite previous `assignedTo` to use `elem` like you did